### PR TITLE
chore(ci): add release pipeline and documentation

### PR DESCRIPTION
**Pull request by sidvishnoi**
## Context

Closes https://github.com/interledger/web-monetization-extension/issues/467

### CODE REVIEW

**Comment by raducristianpopa** [comment]
<h2>Extension builds preview</h2>

<table role="table">
<thead>
  <tr>
    <th>Name</th>
    <th>Link</th>
  </tr>
</thead>
<tbody>
  <tr>
    <td align="center">Latest commit</td>
    <td align="center">7a68ab349c8602dc6947d852c7c72daa9068c27e</td>
  </tr>
  <tr>
    <td align="center">Latest job logs</td>
    <td align="center"><a href="https://github.com/interledger/web-monetization-extension/actions/runs/10450834393">Run #10450834393</a></td>
  </tr>
  <tr><td align="center">Badge</td><td align="center"><a href="https://github.com/interledger/web-monetization-extension/suites/27323496979/artifacts/1826680977">Download</a></td></tr><tr><td align="center">Badge</td><td align="center"><a href="https://github.com/interledger/web-monetization-extension/suites/27323496979/artifacts/1826680981">Download</a></td></tr>
</tbody>
</table>

File: .github/actions/bump-manifest-version.cjs [hunk 1]
```diff
@@ -0,0 +1,47 @@
+// @ts-check
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable no-console */
```
Comment (sidvishnoi) [inline .github/actions/bump-manifest-version.cjs:3]:
```suggestion
/* eslint-disable @typescript-eslint/no-var-requires, no-console */
```

File: .github/actions/bump-manifest-version.cjs [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+/**
+ * @typedef {'build' | 'patch' | 'minor'} BumpType
+ * @param {string} existingVersion
+ * @param {BumpType} type
+ * @return {[major: number, minor: number, patch: number, build: number]}
+ */
+function bumpVersion(existingVersion, type) {
+  const [major, minor, patch, build] = existingVersion.split('.').map(Number)
```
Comment (sidvishnoi) [inline .github/actions/bump-manifest-version.cjs:34]:
```suggestion
  const parts = existingVersion.split('.').map(Number)
  if (parts.length !== 4 || parts.some(e => !Number.isSafeInteger(e))) {
    throw new Error("Existing version does not have right format")
  }
  const [major, minor, patch, build] = parts
```

**Review by raducristianpopa — APPROVED** [review]
Done during pair-programming.

**Pull request merged**